docs/assets/js/sidebar.js

```diff
@@ -5,15 +5,26 @@
 function init() {
   const secondaryNavCat = document.querySelector( '.ds-nav-container' );
 
+  // First collapse the navigation if in mobile.
+  if ( innerWidth < 601 ) {
+    secondaryNavCat.removeAttribute( 'open' );
+  }
+
+  let windowWidth = window.innerWidth;
   /**
    * Test the viewport size and set whether the test passes on the instance.
    */
   function handleViewportChange() {
-    if ( window.innerWidth < 601 ) {
-      secondaryNavCat.removeAttribute( 'open' );
-
-    } else {
-      secondaryNavCat.setAttribute( 'open', 'open' );
+    // Collapse the navigation if we resize to mobile,
+    // but only if we haven't already.
+    // Otherwise, we're on desktop size, so open the navigation.
+    let innerWidth = window.innerWidth;
+    if ( innerWidth !== windowWidth ) {
```

Comment:
I'd vote for either returning early:

```
if ( innerWidth === windowWidth ) {
  return;
}
```

or combining the conditionals:

```
if ( innerWidth !== windowWidth && innerWidth < 601 ) {
  secondaryNavCat.removeAttribute( 'open' );
} else {
  secondaryNavCat.setAttribute( 'open', 'open' );
}
```

Reply:
I went with the first, as the second version would run the `else` in cases where the original structure wouldn't. https://github.com/cfpb/design-system/pull/949/commits/e8d367df2ea004033e59632c3ff4d8c8d91bb9cf